src/App/Examples/web/customers/index.js

```diff
@@ -0,0 +1,743 @@
+    customersDetailedOrdersSteps,
+    customersDetailedLatestInquiry,
+    customersDetailedPreviousInquiries
+} from "./constants";
+
+const { actionList, datepicker, dialog, tabs } = window.dg;
+
+const createNumArray = (length, baseNum, addNum) => (
+    [...Array(length)].map(() => (
+        Math.floor(Math.random() * addNum) + Math.floor(Math.random() * addNum) + Math.floor(Math.random() * addNum) +
+        Math.floor(Math.random() * addNum) + Math.floor(Math.random() * addNum) + baseNum)
+    ) // The multiple Math.floor(Math.random) is there to achieve a lower variance
+);
+
+class CustomersOverview extends Component {
+    constructor (props) {
+        super(props);
+        this.state = {
+            customerFilter: ""
+        };
+    }
+
+    componentDidUpdate () {
+        this.customerFilterField.focus();
+    }
+
+    updateFilter (e) {
+        this.setState({
+            customerFilter: e.target.value.toLowerCase()
+        });
+    }
+
+    resetFilter () {
+        this.setState({
+            customerFilter: ""
+        });
+    }
+
+    render () {
+        return (
+            <>
+                <h2 id="customers-overview">Customers overview</h2>
+                <ComponentPreview language="html" showCasePanel codeFigure>
+                    <div className="panel panel-brand">
+                        <header>
+                            <div className="row">
+                                <div className="col-lg-4">
+                                    <h2 className="panel-title">Customers</h2>
+                                </div>
+                                <div className="col-lg-8">
+                                    <div className="input-group">{"\n"}
+                                        <span className="input-group-addon">{"\n"}
+                                            <i className="material-icons">search</i>
+                                        </span>{"\n"}
+                                        <input ref={input => this.customerFilterField = input} value={this.state.customerFilter} onChange={e => this.updateFilter(e)} type="text" className="form-control" />{"\n"}
+                                        <button type="button" onClick={() => this.resetFilter()} className="btn btn-executive">{"\n"}
+                                            <i className="material-icons">close</i>{"\n"}
+                                        </button>{"\n"}
+                                    </div>
+                                </div>
+                            </div>
+                        </header>
+                        <div className="panel-table">
+                            <table className="table table-hover table-striped">
+                                <thead>
+                                    <tr>
+                                        <th scope="col">ID</th>
+                                        <th scope="col" className="d-none d-sm-table-cell">First name</th>
```

Comment:
What is the reason for `d-none d-sm-table-cell`?

Reply:
This is just me playing with ways to present the table on smaller devices. Some of the columns are hidden when the screen is too narrow to show all of the content

Reply:
aah, I see.